.circleci/config.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       - run:
           name: Lint Client and Server
           command: |
-            ./node_modules/.bin/eslint-eslines $(circleci tests glob "client/**/*.js" "client/**/*.jsx" "server/**/*.js" "server/**/*.jsx" | circleci tests split)
```

Comment:
I'd consider splitting the greps, simpler and less wild-card matching. Either way, it would be good to anchor `\.jsx?` to end with `$`.

```sh
./node_modules/.bin/eslint-eslines $(git diff --name-only origin/master... | grep -E '^(client|server)/' | grep -E '\.jsx?$' | circleci tests split)
```